presets/4.3/vtx/speedybee_tx800.txt

```diff
@@ -0,0 +1,51 @@
+#$ TITLE: SpeedyBee TX800 VTX Tables
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS:  vtx, vtx table, speedybee, tx800
+#$ AUTHOR: NemesisXB
+#$ DESCRIPTION: VTX tables for the SpeedyBee TX800.
+#$ DESCRIPTION: Select the regulatory domain in the options (EU or US).
+#$ DISCLAIMER:  Only select one regulatory option. All previous VTX Table settings will be reset.
+#$ INCLUDE_DISCLAIMER: misc/disclaimer/en/vtxtable.txt
+#$ DISCUSSION:  https://github.com/betaflight/firmware-presets/pull/105
+
+# reset basic Vtx settings
```

Comment:
better to use this `#$ INCLUDE: presets/4.3/vtx/defaults.txt`
instead of manually reseting thing. just to keep it tidy and constant with other tables

Reply:
This also not only clears the vtx table but also the vtx settings. I Don't think this should.